### DISCUSSION TITLE
Upgrade MySQL version in running container?

### DISCUSSION THREAD

**Discussion by ulfie22** [tighten/takeout#272] [Q&A]
Hi Tighten/Takeout -

I am running a container which has MySQL 8.0.21 - I want to upgrade MySQL to the latest so I can take advantage of "invisible" columns. Is there a way to do this through Takeout?

Thanks!!

**Comment by josecanhelp** [comment]
@ulfie22 containers aren't really meant to be upgraded. However, you can try exporting your data to sql dump file and bringing up a new container and importing the dump.

**Reply by ulfie22** [reply]
Thanks, Jose - I am trying that.
